docs/UserGuide.adoc

```diff
@@ -215,12 +276,50 @@ The `redo` command fails as there are no `undo` commands executed previously.
 === Clearing all entries : `clear`
 
 Clears all entries from the address book. +
-Format: `clear`
+Format: `clear` +
+Alias: `c`
+
+=== Access a person's Facebook : `facebook`
```

Comment:
I'm wondering if we should abstract this instead to `socialmedia 1 facebook` for example. This way we can easily add more social media integrations. What do you think?

Reply:
Easier to implement + less copy-pasting as well.

Reply:
Hmmm because I was trying to make it consistent with the add command, i.e. each social media is a prefix on its own. But I guess we can implement your suggestion instead to allow easier expansion to other social media in the future.

Reply:
New commit to reflect changes :)